examples/crm/src/dataProvider.ts

```diff
@@ -89,7 +89,7 @@ const dataProviderWithCustomMethod = {
         }
 
         const sale = sales.data.find(sale => sale.email === email);
-        if (!sale) {
+        if (!sale || sale.disabled) {
```

Comment:
**note:** We will need to add an error message here if the account is disbaled in the supabase version